packages/libs/error-stack/src/lib.rs

```diff
@@ -445,12 +450,13 @@ mod report;
 
 mod context;
 mod ext;
-#[cfg(feature = "hooks")]
+pub mod fmt;
+#[cfg(feature = "std")]
 mod hook;
 
 #[doc(inline)]
 pub use self::ext::*;
-#[cfg(feature = "hooks")]
+#[cfg(feature = "std")]
 pub use self::hook::HookAlreadySet;
```

Comment:
Do we want to deprecate this type as well or is it needed for the new hook-API somewhere?

Reply:
Can safely deprecate.